### ISSUE TITLE
TV series support.

### ISSUE THREAD

**Issue by aandis** [osiro/enceladus#4]
Great lib. Any plans of adding tv series support in the future?

**Comment by osiro** [comment]
Yeah, just running out of time atm, have a look at [movie.rb](https://github.com/osiro/enceladus/blob/master/lib/enceladus/models/movie.rb) and [movie_collection.rb](https://github.com/osiro/enceladus/blob/master/lib/enceladus/models/movie_collection.rb) if you have a chance, implementation of TV Series might be quite similar to those ones.

Before adding support to other types of resources, I'm planning on double checking the current functionalities, tmdb has done a bunch of api changes since I released the gem, and I wanna be sure everything is gonna work smoothly.

**Comment by aandis** [comment]
Good to know! Will take a look for sure. :)

**Comment by Rtransat** [comment]
Any news about TV features ?

**Comment by osiro** [comment]
Hey @Rtransat I'm working on it.

I'll be pushing a few commits into `develop` branch along the next weeks. Will give you a hoy when I get some stuff done.
